src/docs/guides/play.md

```diff
@@ -0,0 +1,321 @@
+---
+title: Deploy a Scala Play App
+---
+
+[Play](https://www.playframework.co) is a high velocity and productive web framework for Java and Scala. It is based on a lightweight, stateless, web-friendly architecture and features predictable and minimal resource consumption (CPU, memory, threads) for highly-scalable applications thanks to its reactive model, based on Pekko Streams.
+
+This guide covers how to deploy a Scala Play app to Railway in four ways:
+
+1. [One-click deploy from a template](#one-click-deploy-from-a-template).
+2. [From a GitHub repository](#deploy-from-a-github-repo).
+3. [Using the CLI](#deploy-from-the-cli).
+4. [Using a Dockerfile](#use-a-dockerfile).
+
+Now, let's create a Play app!
+
+## Create a Play App
+
+**Note:** If you already have a Play app locally or on GitHub, you can skip this step and go straight to the [Deploy Play App to Railway](#deploy-play-app-to-railway).
```

Comment:
the hash should be `#deploy-the-play-app-to-railway`

Reply:
Good catch!